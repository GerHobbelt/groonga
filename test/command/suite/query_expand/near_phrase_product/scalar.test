table_create Terms TABLE_PAT_KEY ShortText --normalizers NormalizerNFKC150
column_create Terms expansion COLUMN_SCALAR ShortText

load --table Terms
[
{"_key": "PGroonga", "expansion": "PGroonga \"Groonga PostgreSQL\""},
{"_key": "Rroonga",  "expansion": "Ruby \"Ruby Groonga\""}
]

query_expand \
  Terms.expansion \
  '*NPP"(Groonga) (Rroonga) (Mroonga)"'
