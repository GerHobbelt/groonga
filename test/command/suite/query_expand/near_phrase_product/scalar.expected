table_create Terms TABLE_PAT_KEY ShortText --normalizers NormalizerNFKC150
[[0,0.0,0.0],true]
column_create Terms expansion COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
load --table Terms
[
{"_key": "PGroonga", "expansion": "PGroonga \"Groonga PostgreSQL\""},
{"_key": "Rroonga",  "expansion": "Ruby \"Ruby Groonga\""}
]
[[0,0.0,0.0],2]
query_expand   Terms.expansion   '*NPP"(Groonga) (Rroonga) (Mroonga)"'
[[0,0.0,0.0],"*NPP\"(Groonga) (Ruby \\\"Ruby Groonga\\\") (Mroonga)\""]
